soroban-env-host/src/host/declared_size.rs

```diff
@@ -278,17 +275,17 @@ mod test {
         // xdr types
         expect!["8"].assert_eq(size_of::<TimePoint>().to_string().as_str());
         expect!["8"].assert_eq(size_of::<Duration>().to_string().as_str());
-        expect!["40"].assert_eq(size_of::<ScVal>().to_string().as_str());
-        expect!["40"].assert_eq(size_of::<ScValObject>().to_string().as_str());
-        expect!["80"].assert_eq(size_of::<ScMapEntry>().to_string().as_str());
+        expect!["64"].assert_eq(size_of::<ScVal>().to_string().as_str());
```

Comment:
Is ScVal bigger due to the instance?

Reply:
Yes, this is an unfortunate side effect of including it in `ScVal`. FWIW the only reason for it to be `ScVal` in the first place is to be able to store it in `ContractData` LE - but probably it's too late to change its structure.

Reply:
Yeah that's not great, but I agree too late to mess with this now.